src/Lean/Elab/BuiltinCommand.lean

```diff
@@ -118,6 +118,50 @@ private partial def elabChoiceAux (cmds : Array Syntax) (i : Nat) : CommandElabM
 @[builtin_command_elab choice] def elabChoice : CommandElab := fun stx =>
   elabChoiceAux stx.getArgs 0
 
+/-- Declares one or more universe variable(s).
+
+Type universes are used in `Type u` and `Sort u` types. While Lean mostly handles universes
+automatically, explicitely declaring some eases the process of writing signatures and type
+definitions by giving us more control and factoring type universe declarations.
```

Comment:
We talked about this wording in the Zulip thread - it is still difficult for me to parse it. What about the suggestions we made over there?

Reply:
My bad, I thought I pushed this already.